src/main/kotlin/racing/RaceExecutor.kt

```diff
@@ -0,0 +1,29 @@
+package racing
+
+object RaceExecutor {
+    private const val CAR_RACE_FORWARD_CONDITION = 4
+
+    fun prepareCar(carCount: Int): List<Car> {
+        val cars: MutableList<Car> = mutableListOf()
+        for (i in 1..carCount) {
+            cars.add(Car(""))
+        }
+        return cars
+    }
+
+    fun raceCarRandomly(
+        cars: List<Car>,
+        tryCount: Int,
+        randomValue: () -> Int = { RandomGenerator.getRandomValue() }
+    ) {
+        for (i in 1..tryCount) {
+            for (car in cars) {
+                if (movable(randomValue)) {
```

Comment:
movable 내부의 로직 정도는 굳이 메서드로 쪼갤 필요 없이 지금처럼 사용해도 가독성이 좋은 것 같아요. 😉
오히려 불필요하게 메서드를 쪼개게면 메서드의 세부 구현을 확인하기위해 화면을 움직이게 되고 이것이 더 비횰유적이라고 생각해요. 🤔

```suggestion
                if (randomValue() >= CAR_RACE_FORWARD_CONDITION) {
```